Azure/azure-cli
# [Keyvault] `az keyvault key create`: Support OKP key and Ed25519 curve

**Related command**

`az keyvault key create`

**Description**

Bump `azure-keyvault-keys` to use `7.4-preview.1` API version
`az keyvault key create`:
- `--kty` support new key type: `OKP` and `OKP-HSM`
- `--curve` support new Elliptic curve `Ed25519`

**Testing Guide**

`az keyvault key create --hsm-name $HSM --name testokpkey --kty OKP-HSM --curve Ed25519`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
